storybookjs/storybook
# CLI: Change Button stories layout for React starter templates

Relates to #22784



## What I did

- Add `layout: 'centered'` to button stories
- Add Typescript 4.9 templates to @storybook/nextjs

## How to test

1. Run a sandbox for a react template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Check the example stories

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
